src/microbe_stage/ProcessList.cs

```diff
@@ -33,6 +35,21 @@ public IEnumerable<IProcessDisplayInfo>? ProcessesToShow
     /// </summary>
     public LabelSettings? ProcessesTitleColour { get; set; }
 
+    /// <summary>
+    ///   External, more technical modifiers for speed, like 2x gameplay speed modifier
+    /// </summary>
+    public float ExternalSpeedModifier
+    {
+        get => externalSpeedModifier;
+
+        set
+        {
+            externalSpeedModifier = value;
```

Comment:
This should have an optimization where if the modifier didn't change this just returns and doesn't call `UpdateEquationsExternalSpeedModifier`.